app/javascript/components/components/Header.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import { BrowserRouter as  Router, Route, Switch, NavLink } from "react-router-dom"
+import { Nav, NavItem } from "reactstrap"
+import Home from "./pages/Home"
+import AboutUs from "./pages/AboutUs"
+import LearnMore from "./pages/LearnMore"
+
+
+class Header extends Component {
+  render(){
+    const {
+      logged_in,
+      current_user,
+      new_user_route,
+      sign_in_route,
+      sign_out_route
+    } = this.props
+
+    return(
+      <>
+      <h1>This is a header</h1>
+      {
+          logged_in &&
+            <div>
+              <a href={sign_out_route}>Sign Out</a>
+            </div>
+        }
+        {
+          !logged_in &&
+            <div>
+              <a href={sign_in_route}>Sign In</a>
+            </div>
+        }
+        <Router>
```

Comment:
You should only have one router per application and your router is in App.js. This is also code that you should not have in this application as this is not from the apartment app project.

Reply:
I removed it